### PR TITLE
[mio-tflite280] Support ruy library

### PR DESCRIPTION
**Pull request by jinevening**
This supports ruy in mio-tflite280.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/issues/10066
Draft PR: #10081

### CODE REVIEW

**Comment by seanshpark** [comment]
@jinevening , have you checked this in Ubuntu 20.04 and Ubuntu 22.04?

**Comment by seanshpark** [comment]
from #10020 , I've disabled using RUY in ARM cross build.
I don't think this change is for x86-64 host.
What target is this for?

**Comment by jinevening** [comment]
The change is for one-optimize, so the main target is x86 (I have no idea on-device compiler will have this code).

If ruy is not allowed, I'll find another kernel (e.g., legacy kernel using Eigen).

**Comment by seanshpark** [comment]
> so the main target is x86 

Ah, this is for x86-64... I think disabling for ARM cross build has nothing to do with this.

File: compiler/mio-tflite280/CMakeLists.txt [hunk 1]
```diff
@@ -44,9 +44,16 @@ if(NOT TensorFlowGEMMLowpSource_FOUND)
...
+nnas_find_package(TensorFlowRuySource EXACT 2.8.0 QUIET)
+
+if(NOT TensorFlowRuySource_FOUND)
+  return()
+endif(NOT TensorFlowRuySource_FOUND)
+
 add_library(mio_tflite280_inc INTERFACE)
 target_include_directories(mio_tflite280_inc SYSTEM INTERFACE "${TensorFlowSource_DIR}")
 target_include_directories(mio_tflite280_inc SYSTEM INTERFACE "${TensorFlowGEMMLowpSource_DIR}")
+target_include_directories(mio_tflite280_inc SYSTEM INTERFACE "${TensorFlowRuySource_DIR}")
```
Comment (seanshpark) [inline compiler/mio-tflite280/CMakeLists.txt:56]: Could you please explain why you added RUY header includes to `mio-tflite` ?
I could guess but this module is for IO of tflite files not kernels.

Comment (jinevening) [reply compiler/mio-tflite280/CMakeLists.txt:56]: I found that `FoldDepthwiseConv2DPass` in `luci_pass` is using `mio-tflite` which includes `GEMMLowpSource`.

Similarly, I thought that `FoldFullyConnectedPass` can use `mio-tflite` which includes `Ruy`.

Comment (seanshpark) [reply compiler/mio-tflite280/CMakeLists.txt:56]: I'm not sure but I think GEMMLowpSource was added from dependency to use tflite source codes.

> Similarly, I thought that FoldFullyConnectedPass can use mio-tflite which includes Ruy.

Yes, logically this is OK.
The thing that makes me hesitate is that `mio-tflite` is getting complicated.
I'll post an issue to separate things.

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**